src/lib/handleCustomEditsRemoval.ts

```diff
@@ -0,0 +1,59 @@
+import { FileName } from "@/components/package-port/CodeEditorHeader"
+
+export const handleCustomEditsRemoval = (
+  files: Record<string, string>,
+  updateFileContent: (filename: FileName, content: string) => void,
+  toast: (options: {
+    title: string
+    description: string
+    variant?: "default" | "destructive"
+  }) => void,
+  entrypointFileName: FileName = "index.tsx",
+  isSchematicWarning: boolean = false,
+) => {
+  try {
```

Comment:
Too much stuff happening inside the try block, if possible don't use it. Or just shorten to specific thing you want to handle inside it.